GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -114,35 +114,20 @@ protected override void Execute(GVFSEnlistment enlistment)
 
                 if (!GVFSPlatform.Instance.KernelDriver.IsReady(tracer, enlistment.EnlistmentRoot, this.Output, out errorMessage))
                 {
-                    if (GVFSPlatform.Instance.UnderConstruction.SupportsGVFSService)
-                    {
-                        tracer.RelatedEvent(
-                            EventLevel.Informational,
-                            $"{nameof(MountVerb)}_{nameof(this.Execute)}_EnablingKernelDriverViaService",
-                            new EventMetadata
-                            {
-                                { "KernelDriver.IsReady_Error", errorMessage },
-                                { TracingConstants.MessageKey.InfoMessage, "Service will retry" }
-                            });
-
-                        if (!this.ShowStatusWhileRunning(
-                            () => { return this.TryEnableAndAttachPrjFltThroughService(enlistment.EnlistmentRoot, out errorMessage); },
-                            $"Attaching ProjFS to volume"))
+                    tracer.RelatedEvent(
+                        EventLevel.Informational,
+                        $"{nameof(MountVerb)}_{nameof(this.Execute)}_EnablingKernelDriverViaService",
+                        new EventMetadata
                         {
-                            this.ReportErrorAndExit(tracer, ReturnCode.FilterError, errorMessage);
-                        }
-                    }
-                    else
-                    {
-                        tracer.RelatedEvent(
-                            EventLevel.Informational,
-                            $"{nameof(MountVerb)}_{nameof(this.Execute)}",
-                            new EventMetadata
-                            {
-                                { "KernelDriver.IsReady_Error", errorMessage },
-                            });
-
-                        this.ReportErrorAndExit(tracer, ReturnCode.FilterError, errorMessage);
+                            { "KernelDriver.IsReady_Error", errorMessage },
+                            { TracingConstants.MessageKey.InfoMessage, "Service will retry" }
+                        });
+
+                    if (!this.ShowStatusWhileRunning(
+                        () => { return this.TryEnableAndAttachPrjFltThroughService(enlistment.EnlistmentRoot, out errorMessage); },
```

Comment:
Is this going to run on Mac?

Reply:
Yes. This is how it looks today (when Kext is not loaded and user types in wrong password 3 times) - 

```
$ gvfs clone https://mseng@dev.azure.com/mseng/AzureDevOps/_git/GVFS GVFS_1
Clone parameters:
  Repo URL:     https://mseng@dev.azure.com/mseng/AzureDevOps/_git/GVFS
  Branch:       Default
  Cache Server: Default
  Local Cache:  /Users/ameen/.gvfsCache
  Destination:  /Users/ameen/Work/Microsoft/VFSForGit/TestRepos/GVFS_1
Authenticating...Succeeded
Querying remote for config...Succeeded
Using cache server: None (https://mseng@dev.azure.com/mseng/AzureDevOps/_git/GVFS)
Cloning...|PrjFS_ConvertDirectoryToVirtualizationRoot(/Users/ameen/Work/Microsoft/VFSForGit/TestRepos/GVFS_1/src)
Cloning...Succeeded
Fetching commits and trees from origin (no cache server)...Succeeded
Driver not loaded.  Attempting to load. You may be prompted for sudo password...
Password:
Password:
Password:
Attaching ProjFS to volume...Failed. Run 'gvfs log /Users/ameen/Work/Microsoft/VFSForGit/TestRepos/GVFS_1' for more info.
GVFS.Service responded with unexpected message: UnknownRequest
```

Reply:
Updated behavior - on the Mac `gvfs mount` will Not call `TryEnableAndAttachPrjFltThroughService`.


```
$ gvfs mount
Driver not loaded.  Attempting to load. You may be prompted for sudo password...
Password:
Password:
Password:
org.vfsforgit.PrjFSKext is not loaded. Make sure the kext is loaded and try again.
```